docs/testing/troubleshooting.md

```diff
@@ -86,6 +86,14 @@ The following steps should help troubleshoot any problems related to the **Autom
 > [!NOTE]
 > There is a known issue that prevents scripts stored in SharePoint from always appearing in the recently used list. This occurs when your admin turns off Exchange Web Services (EWS). Your SharePoint-based scripts are still accessible and usable through the file dialog.
 
+## Problems scheduling a script
+
+Scheduling a script requires a business license of Microsoft 365. This is needed for the Power Automate flow that powers the scheduled script. Errors from this flow are shown when there are problems scheduling. You may need to manually update your flow in [Power Automate](https://make.powerautomate.com), where it will be listed under **My flows**.
```

Comment:
I think we usually use this preposition for licenses: 
```suggestion
Scheduling a script requires a business license for Microsoft 365. This is needed for the Power Automate flow that powers the scheduled script. Errors from this flow are shown when there are problems scheduling. You may need to manually update your flow in [Power Automate](https://make.powerautomate.com), where it will be listed under **My flows**.
```